resources/EPvalidator.rb

```diff
@@ -49,8 +49,7 @@ def self.run_validator(hpxml_doc)
 
         "/HPXML/Building/BuildingDetails/ClimateandRiskZones/WeatherStation" => one, # See [WeatherStation]
 
-        "/HPXML/Building/BuildingDetails/Enclosure/AirInfiltration[AirInfiltrationMeasurement[HousePressure=50]/BuildingAirLeakage[UnitofMeasure='ACH' or UnitofMeasure='CFM']/AirLeakage | AirInfiltrationMeasurement/extension/ConstantACHnatural]" => one, # ACH50, CFM50, or constant nACH; see [AirInfiltration]
-        "/HPXML/Building/BuildingDetails/Enclosure/AirInfiltration/AirInfiltrationMeasurement/InfiltrationVolume" => zero_or_one, # Assumes InfiltrationVolume = ConditionedVolume if not provided
+        "/HPXML/Building/BuildingDetails/Enclosure/AirInfiltration/AirInfiltrationMeasurement" => one, # see [AirInfiltration]
```

Comment:
This change has a subtle difference that is importance. Previously, you could provide an HPXML file that had additional `AirInfiltrationMeasurement` elements beyond what was needed for simulation and they would be ignored. With this change, you would not be able to have extra information stored in the HPXML file since only one `AirInfiltrationMeasurement` element is allowed.

So I think the top line needs to be left where it is (and apply your xpath bugfix there).

Reply:
Isn't only one AirInfultrationMeasurement allowed in HPXML?

Reply:
Sorry the previous comment was deleted because I was thinking about AirInfiltration, which is only allowed to be 1 max occurance. 
You're right, this subtle difference did exist. If I fixed it in the top line, then the check will be across all measurement elements, probably be fine if the information in additional elements is not relevant to our required inputs.